### DISCUSSION TITLE
How to actually install code-server?

### DISCUSSION THREAD

**Discussion by Chainsburg** [coder/code-server#4784] [Q&A] [answered]
Hello,
At this point I've tried several guides and they have either been very outdated or incomplete.
Can you please link a full up-to-date installation process?
Thank you so much!

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
The installation is pretty simple. In the README, it mentioned that you need Linux with a minimum of 1-2 core. Once you got that, run this on the terminal:
```
curl -fsSL https://code-server.dev/install.sh | sudo bash
```
If you are on macOS, install via HomeBrew:
```
brew install code-server
```
Starting it up is simple. Just run `code-server`. For upgrading to the latest version, run the same commands, it will replace the old files.
If you are on Windows, get WSL and run the same Linux command above.
PS: Linux requirements: Linux machine with WebSockets enabled, 1 GB RAM, and 2 CPUs

However, if you are installing on Debian/Ubuntu or RHEL-based Linux distros, I recommend you to go to https://github.com/coder/code-server/releases/latest and getting the binary. Then executing that with APT or RPM, depending on the distro you chose.

**Reply by Chainsburg** [reply]
Thanks a ton! I started a fresh instance and this worked like a charm.

**Reply by jsjoeio** [reply]
You can always consult the [official docs](https://coder.com/docs/code-server/latest/install) if you get stuck!
